nservicebus/azure/publisher-names-configuration.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Publishers name configuration
+summary: Configuration mapping between publisher names and message types for Endpoint Oriented Topology
+tags:
+- Cloud
+- Azure
+- Transports
+---
+
+When Endpoint Oriented Topology is selected, mapping between publisher names and message types has to be properly configured at transport level to ensure that message routing works as expected.  
+Mapping can be configured once message type at time:
+
+snippet: publisher_names_mapping_by_message_type
+
+or supplying an assembly to scan. Configured `Conventions` will be used to extract messages:
+
+snippet: publisher_names_mapping_by_assembly
+
+_publisherName_ is the name of the endpoint that will deliver mapped message.
+  
+`ITopology` interface exposes a property (`NeedsMappingConfigurationBetweenPublishersAndEventTypes`) to instruct transport environment if mapping must be configured. Custom topologies have to properly configure this property.   
```

Comment:
Will there be an article related to "Creation custom topologies"? I don't know. As you said it won't be a common scenario